hyperdome_client/hyperdome_client.py

```diff
@@ -218,7 +228,20 @@ def on_history_added(self):
         self.chat_history = []
 
     def get_uid(self):
-        self.uid = session.get(self.server.url + '/generate_guest_id').text
+        try:
+            self.uid = self.session.get(self.server.url + '/generate_guest_id').text
+        except Exception as e:
+            raise e
```

Comment:
whAT
I 
don't understand the point of catching and then immediately re-raising

Reply:
because I deal with it in the calling funtion, as well as other exceptions that might happen?

Reply:
But how is that any different than just removing the try and except? Without it, the exception will still be raised and caught in the calling function